nbodykit/tests/test_lab.py

```diff
@@ -51,7 +48,7 @@ def test_paint(comm):
     output = "./test_paint-%d.bigfile" % comm.size
     alg.result.save(output)
 
-@MPIWorld(NTask=[2, 3, 4])
+@MPIWorld(NTask=[2, 3, 4], required=2)
```

Comment:
@rainwoodman -- without ``required=2`` this test appears to get run with NTask=1, must be a bug in mpi4py_test